acceptance/utilities_test.go

```diff
@@ -94,6 +96,25 @@ func makeApp(appName string) string {
 	return pushOutput
 }
 
+func makeApp2(appName string) string {
```

Comment:
We could make the makeApp method more flexible instead of duplicating it but I am fine with the current implementation.

Reply:
Things, we only need these two variants. As the second is used only once I could inline it into the test.